src/org/opendatakit/briefcase/ui/reused/SourceDisplayPanelForm.java

```diff
@@ -33,11 +34,16 @@ public void onReset(Runnable callback) {
     onResetCallbacks.add(callback);
   }
 
-  public void readConfiguration(AggregateServerConnectionConfiguration conf) {
+  public void readConfigAggregate(AggregateServerConnectionConfiguration conf) {
```

Comment:
You could give this method and the next the same name `readConfiguration` and let the overload work out which one must run...

Reply:
Ok :)